core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -265,6 +265,22 @@ public String changePrimaryGateway(Stack stack) throws CloudbreakException {
         }
     }
 
+    public void redeployGatewayCertificate(@Nonnull Stack stack, @Nonnull Cluster cluster, List<String> candidateAddresses) {
```

Comment:
Where do you use the `candidateAddresses` param?

Reply:
Nowhere, so removing it. Thanks! 👍